adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -85,6 +85,16 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var coalesce: Int = -1
   @Args4jOption(required = false, name = "-sort_fastq_output", usage = "Sets whether to sort the FASTQ output, if saving as FASTQ. False by default. Ignored if not saving as FASTQ.")
   var sortFastqOutput: Boolean = false
+  @Args4jOption(required = false, name = "-force_load_bam", usage = "Forces Transform to load from BAM.")
+  var forceLoadBam: Boolean = false
+  @Args4jOption(required = false, name = "-force_load_sam", usage = "Forces Transform to load from SAM.")
+  var forceLoadSam: Boolean = false
+  @Args4jOption(required = false, name = "-force_load_fastq", usage = "Forces Transform to load from unpaired FASTQ.")
+  var forceLoadFastq: Boolean = false
+  @Args4jOption(required = false, name = "-force_load_fastq", usage = "Forces Transform to load from interleaved FASTQ.")
+  var forceLoadIFastq: Boolean = false
```

Comment:
There was a missing `i --> interleaved`; I had fixed it locally but forgot to commit before pushing.